Submitted: Dominique Leuenberger (dimstar) - 2011-12-01 03:38:02 PST

Since glib 2.31, including any header than glib.h (plus some exceptions) is
no longer possible. Headers are indirectly implicit included by glib.h

The attached patch fixes what's needed.

http://sourceforge.net/tracker/?func=detail&aid=3446993&group_id=192483&atid=941576

Index: tsclient-2.0.2/src/tsc-bookmarkfile.h
===================================================================
--- tsclient-2.0.2.orig/src/tsc-bookmarkfile.h
+++ tsclient-2.0.2/src/tsc-bookmarkfile.h
@@ -21,7 +21,6 @@
 #define __TSC_BOOKMARK_FILE_H__
 
 #include <glib.h>
-#include <glib/gerror.h>
 #include <time.h>
 
 G_BEGIN_DECLS
